### PR TITLE
Add MWS Modules

### PR DESCRIPTION
**Pull request by bluemellophone**
## Pull Request Overview

This PR adds the (first round) of required modules for the MWS project:

- Adds the `Mission` and `Task` models
- Adds "tags" to `Assets` to mirror the Annotation Keywords
- Adds `contributor` user column to `Annotations`
- Adds a new `Interpreter` user role
- Miscellaneous linting, bug fixes for the slimmed down MWS module config
- Bumps the MWS frontend repository
- Database migration `47a80ead3a90 `.

---

**Model Updates** 

**Mission**
- Users
- Assets
- Tasks
- title
- owner
- options
- jobs
- classifications
- notes

**Task**
- Assets
- Annotations
- type
- notes

**User**
- \+ Tasks

**UserRoles**
- \+ Interpreter

**Assets**
- \+ Tags (Keywords)
- \+ lines
- \+ classifications

**Annotations**
- \+ contributor

### CODE REVIEW

File: app/modules/asset_groups/models.py [hunk 1]
```diff
@@ -883,7 +883,9 @@ class AssetGroup(db.Model, HoustonModel):
         cascade='all, delete',
     )
 
-    assets = db.relationship('Asset', back_populates='asset_group', order_by='Asset.guid')
```
Comment (hwindsor) [inline app/modules/asset_groups/models.py:886]: Any reason for this?. I keep meaning to remove that as I don't think it's actually used but not looked into it properly to be sure

File: app/modules/asset_groups/models.py [hunk 2]
```diff
@@ -883,7 +883,9 @@ class AssetGroup(db.Model, HoustonModel):
...
+    # assets = db.relationship(
```
Comment (hwindsor) [inline app/modules/asset_groups/models.py:886]: Curious why this is commented out, that should be used IIRC.

Comment (bluemellophone) [reply app/modules/asset_groups/models.py:886]: It was commented out because the `back_populates` was converted into a `backref` on the AssetGroup model.  The need for this change is so that asset groups can be disabled (not needed for MWS) and the relationship is only defined with annotations when it is enabled.  We need to come up with a standardized way to document backrefs.  The previous attempt to convert these relationships to backrefs left commented out attributes in their old place so we wouldn't forget about it.

This PR may be the right time to figure out how to document these backrefs and the attributes they imply when enabled.

**Review by colinwkingen — APPROVED** [review]
LGTM

File: app/modules/job_control/models.py [hunk 3]
```diff
@@ -2,10 +2,6 @@
 """Provides UI space served directly from this application"""
 import logging
 
-from app.modules.asset_groups.models import AssetGroupSighting
```
Comment (hwindsor) [inline app/modules/job_control/models.py:5]: I understood that the problem that having the imports ithin functiosn was now fixed and they could be at the module level. Why did you move this?
